resources/views/dinnerform/admin_includes/dinnerform-details.blade.php

```diff
@@ -0,0 +1,93 @@
+<form method="post"
+      action="{{ ($dinnerformCurrent == null ? route("dinnerform::add") : route("dinnerform::edit", ['id' => $dinnerformCurrent->id])) }}"
+      enctype="multipart/form-data">
+
+    {!! csrf_field() !!}
+
+    <div class="card mb-3">
+
+        <div class="card-header bg-dark text-white">
+            Dinner form details
+        </div>
+
+        <div class="card-body">
+
+            <div class="row">
+
+                <div class="col-md-6">
+
+                    <div class="row align-items-end mb-6">
+
+                        <div class="col-md-12 mb-3">
+
+                            <label for="restaurant">Dinner form restaurant:</label>
+                            <input type="text" class="form-control" id="restaurant" name="restaurant"
+                                   placeholder="Elat Roma"
+                                   value="{{ $dinnerformCurrent->restaurant or '' }}"
+                                   required>
+
+                        </div>
+                    </div>
+
+                    <div class="row align-items-end mb-6">
+                        <div class="col-md-12 mb-3">
+
+                            <label for="description">Description:</label>
+                            <input type="text" class="form-control" id="description" name="description"
+                                   placeholder="Order with us at Elat Roma" value="{{ $dinnerformCurrent->description or '' }}" required>
+
+                        </div>
+                    </div>
+                    <div class="row align-items-end mb-6">
+                        <div class="col-md-12 mb-3">
+
+                            <label for="url">Url:</label>
+                            <input type="text" class="form-control" id="url" name="url"
+                                   placeholder="https://forms.gle/t2hDEnkNCLXNpvYTA" value="{{ $dinnerformCurrent->url or 'https://forms.gle/t2hDEnkNCLXNpvYTA' }}" required>
```

Comment:
Volgens mij is deze vrij constant. Wat mij betreft zet je 'm standaard in de value, niet de placeholder, dat scheelt weer copypaste.

Reply:
Je hebt deze wel geresolved maar niet doorgevoerd?